front/src/app/pages/convention/ConventionSignPage.tsx

```diff
@@ -190,8 +190,25 @@ const ConventionSignPageContent = ({
                       }
                     />
                   )}
+                  {convention.status === "DEPRECATED" && (
+                    <Alert
+                      severity="error"
+                      title={t.sign.deprecated.title}
+                      description={
+                        <>
+                          <p className={fr.cx("fr-mt-1w")}>
+                            {t.sign.deprecated.detail}
+                          </p>
+                          <p>
+                            Les raisons sont : {convention.statusJustification}
+                          </p>
```

Comment:
on peut éventuellement le mettre en conditionnel, au cas ou la valeur est null.

```suggestion
                          {convention.statusJustification ? <p>
                            Les raisons sont : {convention.statusJustification}
                          </p> : null}
```